input/globals.json

```diff
@@ -37,6 +37,26 @@
       ]
     }
   ],
+  "websites": [
```

Comment:
suggestion: Instead of a synonym, think we should just version it. Say "frontends-v2".